google_guest_agent/agentcrypto/mtls_mds.go

```diff
@@ -267,17 +267,17 @@ func (j *CredsJob) shouldUseNativeStore(mds *metadata.Descriptor) bool {
 	var useNative bool
 
 	if cfg.Get().MDS != nil {
-		useNative = !cfg.Get().MDS.HTTPSMDSSkipNativeStore
+		useNative = cfg.Get().MDS.HTTPSMDSEnableNativeStore
```

Comment:
Is the VM's config file not allowed to enable the native cert store if it's disabled in metadata somewhere?

Reply:
yes, metadata takes higher priority and allows users to configure without being on VM or restarting agent and at scale (can simply set it at project level).